### PR TITLE
Add rustlerbombs and speedometer to default startup resources

### PR DESCRIPTION
**Pull request by F2BShady**
Adds rustlerbombs and speedometer to default startup resources

### CODE REVIEW

**Comment by MegadreamsBE** [comment]
Any issue this resolves specifically? Any reason why those two resources should be run by default? I feel this should have been a discussion point in issues beforehand.

**Comment by F2BShady** [comment]
> Any issue this resolves specifically? Any reason why those two resources should be run by default? I feel this should have been a discussion point in issues beforehand.

it may be needed for speedometer, but I can't say anything about rustlerbombs, the player can activate it at will, it's true what you said, but I think speedomeeter can be a good choice.

**Comment by botder** [comment]
Adding speedometer sounds good, but I don't think we need rustlerbombs

File: Server/mods/deathmatch/mtaserver.conf [hunk 1]
```diff
@@ -284,6 +284,8 @@
     <resource src="resourcebrowser" startup="1" protected="1" default="true"/>
     <resource src="resourcemanager" startup="1" protected="1"/>
     <resource src="scoreboard" startup="1" protected="0"/>
+    <resource src="rustlerbombs" startup="1" protected="0"/>
+    <resource src="speedometer" startup="1" protected="0"/>
```
Comment (botder) [inline Server/mods/deathmatch/mtaserver.conf]:
```suggestion
    <resource src="speedometer" startup="1" protected="0"/>
```

**Review by botder — APPROVED** [review]

**Pull request merged**